.github/workflows/api-pull-request.yml

```diff
@@ -55,7 +55,7 @@ jobs:
               run: black --check .
 
             - name: Check flake8 linting
-              run: flake8 .
+              run: flake8 --config=../.flake8 .
```

Comment:
Why don't we just move the flake8 config file into the api dir?

Reply:
It doesnt play nicely with pre-commit in that folder - spent ages trying to have it work but it wont manage it